### PR TITLE
[BLOCKED]migration: merge vendors and libraries into providers

### PR DESCRIPTION
**Pull request by KonstantinaStoikou**
Closes inveniosoftware/invenio-app-ils#1051

Depends on inveniosoftware/invenio-app-ils#1057

### CODE REVIEW

**Review by ntarocco — COMMENTED** [review]
LGTM! 👏🏼 👏🏼 👏🏼 👏🏼 👏🏼 👏🏼

File: cds_ils/migrator/providers/api.py [hunk 1]
```diff
@@ -41,39 +42,61 @@
...
+        "Found {0} providers with legacy_id {1}".format(
             len(result.hits), legacy_id
         )
     )
 
 
+def get_provider_by_legacy_id(legacy_id):
+    """Search for provider by legacy id."""
+    search = current_ils_prov.provider_search_cls().query(
+        "bool", filter=[Q("term", legacy_ids=legacy_id)]
```
[the remark this answers is not in the document]
Comment (kprzerwa) [reply cds_ils/migrator/providers/api.py:72]: indeed it looks like it is doing the same query

File: ui/package.json [hunk 2]
```diff
@@ -3,7 +3,7 @@
   "version": "1.0.0a33",
   "dependencies": {
     "@babel/runtime": "^7.9.2",
-    "@inveniosoftware/react-invenio-app-ils": "^1.0.0-alpha.28",
+    "@inveniosoftware/react-invenio-app-ils": "^1.0.0-alpha.29",
```
Comment (kprzerwa) [inline ui/package.json:6]: what about package-lock ?

**Review by kprzerwa — COMMENTED** [review]
LGTM!
You are only missing package-lock version bump and .env update

**Review by kprzerwa — APPROVED** [review]

**Review by ntarocco — APPROVED** [review]

**Pull request merged**